### DISCUSSION TITLE
How to train data with question from web ui

### DISCUSSION THREAD

**Discussion by njalan** [vanna-ai/vanna#304] [Q&A]
how to train below question sql pairs from web ui:
vn.train(
    question="What is the average age of our customers?", 
    sql="SELECT AVG(age) FROM customers"
)

**Comment by zainhoda** [comment]
If you click on Training Data -> Add Training Data -> SQL

You can add the SQL statement there. Right now it doesn't accept the associated question, instead it will use the LLM to "infer" the business question from the SQL statement. 

<img width="1242" alt="Screenshot 2024-03-21 at 11 14 00 AM" src="https://github.com/vanna-ai/vanna/assets/7146154/c3d65ae4-0453-4c75-8e91-5384006a07ba">

**Comment by njalan** [comment]
@zainhoda Thanks for your help. Actually I knew these steps and but I only know add sql to train. I mean how to add  question sql pairs  to train on web ui for below, what should I input on web for it
question="What is the average age of our customers?",
sql="SELECT AVG(age) FROM customers"

**Comment by HaoGuo98** [comment]
how to find this page in vanna 2.0?
